services/workers/src/main/java/org/sagebionetworks/report/worker/StorageReportCSVDownloadWorker.java

```diff
@@ -57,20 +55,16 @@ public void run(ProgressCallback progressCallback, Message message) throws Excep
 		AsynchronousJobStatus status = asynchJobStatusManager.lookupJobStatus(message.getBody());
 		String fileName = "Job-"+status.getJobId();
 		File temp = null;
-		CSVWriter writer = null;
-		try{
+		try {
 			UserInfo user = userManager.getUserInfo(status.getStartedByUserId());
 			DownloadStorageReportRequest request = AsynchJobUtils.extractRequestBody(status, DownloadStorageReportRequest.class);
 			// The CSV data will first be written to this file.
 			temp = File.createTempFile(fileName, ".csv");
```

Comment:
Should I use FileProvider? I did not use it here because of mock issues (see below)